FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/workmanager/RefreshClientConfigurationWorker.kt

```diff
@@ -51,7 +54,7 @@ class RefreshClientConfigurationWorker(context: Context, params: WorkerParameter
     val account = roomDatabase.accountDao().getActiveAccountSuspend() ?: return Result.success()
 
     val baseFesUrlPath = GeneralUtil.genBaseFesUrlPath(
-      useCustomerFesUrl = account.useCustomerFesUrl,
+      useCustomerFesUrl = if (isPublicDomain) false else account.useCustomerFesUrl,
```

Comment:
We might simplify as belows
```kt
useCustomerFesUrl = !isPublicDomain && account.useCustomerFesUrl
```

Reply:
Agree. But `if (isPublicDomain) false` looks more explicit. I prefer to leave it as is.

`useCustomerFesUrl = if (isPublicDomain) false else account.useCustomerFesUrl, `- one line
`useCustomerFesUrl = !isPublicDomain && account.useCustomerFesUrl `- one line

Reply:
Ok, As you wish :-)